src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -13,4 +13,20 @@ To see the custom values you used for this release:
 
   $ helm get values {{ .Release.Name }} -n {{ .Release.Namespace }}
 
+{{ if .Values.ingress.create -}}
+Service URL of {{ .Chart.Name }}: {{ ternary "https" "http" .Values.ingress.https -}}://{{ .Values.ingress.host }}
```

Comment:
Very minor but the styling for our product names starts with an uppercase letter. We can use `title` function for this: `{{ title .Chart.Name }}` for all products

https://masterminds.github.io/sprig/strings.html